go/staking/api/api.go

```diff
@@ -24,16 +24,17 @@ const (
 )
 
 var (
-	// CommonPoolAccountID signifies the common pool in staking events.
-	// The ID is invalid to prevent it being accidentally used in the
-	// actual ledger.
-	CommonPoolAccountID = signature.NewBlacklistedKey("1abe11edc001ffffffffffffffffffffffffffffffffffffffffffffffffffff")
-
-	// FeeAccumulatorAccountID signifies the staking fee accumulator in
-	// staking events.
-	// The ID is invalid to prevent it being accidentally used in the
-	// actual ledger.
-	FeeAccumulatorAccountID = signature.NewBlacklistedKey("1abe11edfeeaccffffffffffffffffffffffffffffffffffffffffffffffffff")
+	// CommonPoolAddress signifies the common pool in staking events.
+	// The address is invalid to prevent it being accidentally used in the actual ledger.
+	CommonPoolAddress = NewFromPublicKey(
+		signature.NewBlacklistedKey("1abe11edc001ffffffffffffffffffffffffffffffffffffffffffffffffffff"),
```

Comment:
Consider creating `NewReservedAddress("...hex public key...")` which blacklists the key and returns the appropriate address.

Reply:
Done.